docs/configuration/adding_your_own.md

```diff
@@ -84,55 +68,33 @@ process {
 The variable `wf_container` is defined dynamically and automatically specifies the image tag if Nextflow is running with `-r`.
 
 If you intend to run the pipeline offline, nextflow will not be able to automatically download the singularity image for you. Instead, you'll have to do this yourself manually first, transfer the image file and then point to that.
-
-First, pull the image file where you have an internet connection:
-
-```bash
```

Comment:
Missing the start of this code block, and other markdown weirdness caused by a broken merge.